how-to-guides/quickstart/notebooks/neptune_quickstart.ipynb

```diff
@@ -303,6 +306,78 @@
     "These metrics are also visualized as [Charts](https://docs.neptune.ai/charts/) in the Neptune web app. All charts update live and are visible in the **Charts** tab.\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Log files to Neptune\n",
+    "\n",
+    "You can log various types of files to Neptune:\n",
+    "- Images (PNG, JPEG, etc.)\n",
+    "- Data files (CSV, JSON, etc.) \n",
+    "- Model checkpoints\n",
+    "- And more\n",
+    "\n",
+    "Use the `assign_files()` method to upload files. You can:\n",
+    "- Log single or multiple files in one call\n",
+    "- Organize files in folders\n",
+    "- Add metadata like descriptions or tags\n",
+    "\n",
+    "For more details and examples, see the [file logging docs](https://docs.neptune.ai/log_files)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download a sample image\n",
+    "! curl -o sample.png https://neptune.ai/wp-content/uploads/2024/05/blog_feature_image_046799_8_3_7_3-4.jpg"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run.assign_files({\"files/single_image\": \"sample.png\"})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Log custom string series\n",
+    "\n",
+    "Neptune automatically captures standard console logs (`stdout` and `stderr`) - see docs for capturing [console logs](https://docs.neptune.ai/console_logs).\n",
+    "\n",
+    "You can also log custom string messages using `log_string_series()`. Each message is \n",
+    "associated with a step value, making it useful for tracking progress during training.\n",
+    " \n",
+    "For example, you can log:\n",
+    "- Error messages\n",
+    "- Progress updates \n",
+    "- Custom debugging information\n",
+    "\n",
+    "See [logged example](https://scale.neptune.ai/examples/quickstart/runs/details?viewId=9ea24429-7823-477e-a740-aa667c66a314&detailsTab=dashboard&dashboardId=9ee8a43f-e76d-428d-b2f8-fdec9b2e0670&runIdentificationKey=QUICK-7&type=run&experimentsOnly=false&runsLineage=FULL&lbViewUnpacked=true&sortBy=%5B%22metrics%2Ftest%2Faccuracy%22%5D&sortFieldType=%5B%22floatSeries%22%5D&sortFieldAggregationMode=%5B%22last%22%5D&sortDirection=%5B%22descending%22%5D&suggestionsEnabled=false&query=&experimentOnly=false) in custom dashboard.\n",
+    "\n",
+    "Additionally, see the [docs](https://docs.neptune.ai/log_text) on logging text data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run.log_string_series(\n",
+    "    data={\"custom_messages/errors\": \"Job failed\", \"custom_messages/info\": \"Training completed\"},\n",
+    "    step=5,\n",
```

Comment:
We should log a series of strings to the same attribute across multiple steps to differentiate this from single string (string atom) logging